src/LaboratoryResults-3-0/Test/Receive/2-2a-receive-minimalNHG.xml

```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://hl7.org/fhir/STU3/testscript.sch" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TestScript xmlns="http://hl7.org/fhir" xmlns:nts="http://nictiz.nl/xsl/testscript"
+    nts:scenario="server">
+    <id value="TEST-xis-2-2a-Receive-minimal-NHG"/>
```

Comment:
Consider to rename this scenario '...-receive-singleNHG' (and similar for LOINC), to avoid confusion with the 'min' scenarios.